config.yaml

```diff
@@ -293,34 +293,32 @@ commands:
     - must have ecs:container
   task-list:
     executable: "/home/bundle/cog-command"
-    description: Lists registered tasks.
+    description: Lists task definition families and revisions.
+    long_description: Lists task definition families and revisions. Pass a task definition
```

Comment:
Since these sections are right next to each other in the rendered help docs it might be weird to see the first sentence repeated.